android/app/src/main/AndroidManifest.xml

```diff
@@ -27,6 +27,9 @@
         </intent-filter>
       </activity>
       <activity android:name="com.facebook.react.devsupport.DevSettingsActivity" />
+        <meta-data
+                android:name="com.google.android.geo.API_KEY"
+                android:value=""/>
```

Comment:
Maybe we can get this in js config

Reply:
possible with react-native-config, but not in scope of this task